physical-deployments/physical-deployment.yml

```diff
@@ -4,6 +4,10 @@ internal-iface: eno1
 central-node:
   name: host02.mydomain.com
 
+tester-node:
+  name: host02.mydoman.com
+  ssh_key: id_rsa
```

Comment:
The full path is required here, so it should be better to have something like `/full/path/to/id_rsa` here.  We shouldn't use `/root/.ssh/id_rsa` in the example though to avoid accidental leak of the actual private key.

Reply:
A full path is not actually required here. The reason I have it set to just `id_rsa` is because I generated an SSH key in the same directory as `do.sh`, and I used that key during testing.

However, I think it's a much better example to use a full path instead, so I will change it.

Reply:
I resolved this by using a full path in the config file.